discord-public-channel.cpp

```diff
@@ -32,12 +95,13 @@ pin_message(client *client, const uint64_t channel_id, const uint64_t message_id
     return;
   }
 
-  struct sized_buffer req_body = {"", 0};
+  struct resp_handle resp_handle = {NULL, NULL};
+  struct sized_buffer body = {"", 0};
 
```

Comment:
we deal with both reponse_body and request_body.  Using `req_body` to improves the clarity of the variable.